README.md

```diff
@@ -9,7 +9,7 @@
 [![Gem Version](https://badge.fury.io/rb/sinatra-active-model-serializers.png)](http://badge.fury.io/rb/sinatra-active-model-serializers)
 
 This gem has the function of adapt the Active Model Serializers to work in Sinatra from a more practical way with models.
-If do you use everything at the default, just  require `sinatra_active_model_serializers` for serializers work correctly the a json response.
+If do you use everything at the default, just  require `sinatra_active_model_serializers` for serializers work correctly with a json response.
```

Comment:
@SauloSilva @samuelmadeu 

I think that there are others errors, like:

> If you are using everything by default, just require `sinatra_active_model_serializers` for serializers work correctly with a json response.

What do you think, guys?